### PR TITLE
Increase font size for alarm keypad number buttons

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Allow the option to style the font size for alarm keypad number buttons for better readability. The non-number buttons are left unchanged.

Example when styled to 22px:
grafik

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes https://github.com/home-assistant/frontend/discussions/8005
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by KTibow** [comment]
Maybe this could be in the middle, or a config option? It seems to just not fit in with the rest of the UI.

**Comment by spacegaier** [comment]
@KTibow What do you mean by "be in the middle"?

If we end up not changing it via this PR, then we should perhaps ensure that the keypad number font size can be controlled by a theme, so users have a chance to adjust since I do understand the reasoning for this from a usability point of view..

**Comment by bramkragten** [comment]
I'd rather see this as a themeable option

**Comment by KTibow** [comment]
I mean `21px`.

**Comment by spacegaier** [comment]
> 
> 
> I mean `21px`.

I don't think Material Design allows uneven numbers. But either way, we will not change it directly in the frontend core (see Bram's response), but instead offer the option for themes to define a size. There you could then go for 21px if you wanted to.

File: src/panels/lovelace/cards/hui-alarm-panel-card.ts [hunk 1]
```diff
@@ -364,6 +366,10 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
       mwc-button#disarm {
         color: var(--error-color);
       }
+
+      mwc-button.numberkey {
+        --mdc-typography-button-font-size: var(--keypad-font-size, 0.875rem);
```
Comment (spacegaier) [inline src/panels/lovelace/cards/hui-alarm-panel-card.ts:371]: `0.875rem` is the default from MDC.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**